docs/_layouts/variation.html

```diff
@@ -1,35 +1,11 @@
 ---
 layout: default
 ---
-<main class="content content__1-3 content__bleedbar-expandable" id="main" role="main">
-  <div class="content_bar"></div>
-  <div class="content_wrapper">
-    <aside class="content_sidebar
-                o-expandable
-                o-expandable__padded
-                o-expandable__background
-                o-expandable__border">
-        <button class="o-expandable_header o-expandable_target"
-                title="Expand content">
-            <h3 class="h5 o-expandable_header-left o-expandable_label">
-                Navigation
-            </h3>
-            <span class="o-expandable_header-right o-expandable_link">
-                <span class="o-expandable_cue o-expandable_cue-open">
-                    <span class="u-visually-hidden-on-mobile">Show</span>
-                    {% include icons/plus-round.svg %}
-                </span>
-                <span class="o-expandable_cue o-expandable_cue-close">
-                    <span class="u-visually-hidden-on-mobile">Hide</span>
-                    {% include icons/minus-round.svg %}
-                </span>
-            </span>
-        </button>
-        <div class="o-expandable_content
-                    nav-secondary">
-            {% include sidebar.html %}
-        </div>
-    </aside>
+<main id="main" role="main">
```

Comment:
The GovUK tabs we forked require a parent element to have a `js-enabled` class. We currently add it to the `main` element [here](https://github.com/cfpb/design-system/blob/master/docs/assets/js/main.js#L26). To keep the tabs working, change that selector to `#main` to match this new HTML.

Reply:
Oops, fixed!